### PR TITLE
SD-2853: add docs on how to use XFER

### PR DESCRIPTION
**Pull request by pmcclory-sp**
Document the XFER feature

### CODE REVIEW

File: momentum/4/outbound-xfer.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  Routes = (
+    "xfer://10.10.10.10?port=8080"
+  )
+  HTTP_URI = "/delay_message"
+  HTTP_METHOD = "POST"
+  HTTP_HOST = "localhost"
+}
+```
+
+To reroute messages for speicifc domains to an XFER server configuration like the following is needed:
```
Comment (JohnPeacockMessageSystems) [inline momentum/4/outbound-xfer.md:47]: speling

File: momentum/4/outbound-xfer.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+---
+title: "Configuring XFER for Outbound Mail Delivery"
+description: "By default Momentum will attempt to deliver mail via the SMTP protocol to the destination MX server.  The XFER delivery method can be used to transfer mail over HTTP to another Momentum server that will handle delivery."
+---
+
+## <a name="outbound_mail.XFER"></a> XFER
+
+By default Momentum will attempt to deliver mail via the SMTP protocol to the destination MX server.  The XFER delivery method can be used to transfer mail over HTTP to another Momentum server that will handle delivery.  Some example use cases where this could be useful include transferring tempfailed messages to another server for handling retries, and transferring mail to specific domains to another server that is configured with different settings.
+
+To receive XFER messages from another server configuration like the following is needed:
```
Comment (JohnPeacockMessageSystems) [inline momentum/4/outbound-xfer.md:10]: subheading here for Receiving Server Setup?
```suggestion
### Server Config for Receiving XFer Messages
To receive XFER messages from another server configuration like the following is needed:
```

File: momentum/4/outbound-xfer.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  }
+}
+
+# stand up an HTTP listener on port 8080
+HTTP_Listener {
+  Listen ":8080" {}
+}
+```
+
+To reroute delayed messages to an XFER server configuration like the following is needed (assuming the XFER server is running on 10.10.10.10):
```
Comment (JohnPeacockMessageSystems) [inline momentum/4/outbound-xfer.md:26]:
```suggestion
### Sending Server for XFer Delayed Messages
To reroute delayed messages to an XFER server configuration like the following is needed (assuming the XFER server is running on 10.10.10.10):
```

File: momentum/4/outbound-xfer.md (hunk 1 above)
Comment (JohnPeacockMessageSystems) [inline momentum/4/outbound-xfer.md:47]:
```suggestion
### Sending Server Config for Specific Domains
To reroute messages for specific domains to an XFER server configuration like the following is needed:
```

File: momentum/4/outbound-xfer.md (hunk 2 above)
Comment (mdyer-sparkpost) [inline momentum/4/outbound-xfer.md:10]: Nitpick
```suggestion
To receive XFER messages from another server, configuration like the following is needed:
```

**Review by JohnPeacockMessageSystems — CHANGES_REQUESTED** [review]

File: momentum/4/outbound-xfer.md (hunk 3 above)
Comment (mdyer-sparkpost) [inline momentum/4/outbound-xfer.md:26]: Aanother nitpick
```suggestion
To reroute delayed messages to an XFER server, configuration like the following is needed (assuming the XFER server is running on 10.10.10.10):
```

File: momentum/4/outbound-xfer.md (hunk 1 above)
Comment (mdyer-sparkpost) [inline momentum/4/outbound-xfer.md:47]: Typo fix
```suggestion
To reroute messages for specifc domains to an XFER server, configuration like the following is needed:
```

**Review by mdyer-sparkpost — CHANGES_REQUESTED** [review]

**Review by mdyer-sparkpost — APPROVED** [review]

**Review by JohnPeacockMessageSystems — APPROVED** [review]

**Pull request merged**